components/style/properties/shorthand/mask.mako.rs

```diff
@@ -204,3 +225,85 @@
         }
     }
 </%helpers:shorthand>
+
+<%helpers:shorthand name="mask-position" products="gecko" extra_prefixes="webkit"
```

Comment:
Yes, I missguided you on this. I recommended you to copy-paste this but it would be better to share the logic here too. Sorry about that.